app/templates/tasklist.html

```diff
@@ -11,7 +11,7 @@
 
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
-            <span class="govuk-caption-l">{{ application_meta_data["fund_name"]|kebab_case_to_human }} {{ application_meta_data["round_name"] }}</span>
+            <span class="govuk-caption-l">{{ application_meta_data["fund_name"]}} {{ application_meta_data["round_name"] }}</span>
```

Comment:
good spot! super picky but content brought this up last time I worked on casing, only the fund name is capitalized, not the word round as [here](https://fsd-funding-prototype-v2.herokuapp.com/cof-w2-v2/app-task-list-new#:~:text=Community%20Ownership%20Fund%20round%202)

Reply:
Should we change this upstream in the fund store as the casing comes from there?

Reply:
@Adam-W1 OK have added lower case filter now